src/state/reducer.ts

```diff
@@ -62,49 +54,62 @@ const preferencesPersistConfig = {
 const txHistoryPersistConfig = {
   key: 'txHistory',
   storage: localforage,
-  version: Math.max(...Object.keys(clearTxHistoryMigrations).map(Number)),
-  migrate: createMigrate(clearTxHistoryMigrations, { debug: false }),
+  version: 0,
+  // Uncomment me when introducing the first migration for this slice
+  // version: Math.max(...Object.keys(clearTxHistoryMigrations).map(Number)),
+  // migrate: createMigrate(clearTxHistoryMigrations, { debug: false }),
 }
 
 const portfolioPersistConfig = {
   key: 'portfolio',
   storage: localforage,
-  version: Math.max(...Object.keys(clearPortfolioMigrations).map(Number)),
-  migrate: createMigrate(clearPortfolioMigrations, { debug: false }),
+  version: 0,
+  // Uncomment me when introducing the first migration for this slice
+  // version: Math.max(...Object.keys(clearPortfolioMigrations).map(Number)),
+  // migrate: createMigrate(clearPortfolioMigrations, { debug: false }),
 }
 
 const opportunitiesPersistConfig = {
   key: 'opportunities',
   storage: localforage,
-  version: Math.max(...Object.keys(clearOpportunitiesMigrations).map(Number)),
-  migrate: createMigrate(clearOpportunitiesMigrations, { debug: false }),
+  version: 0,
+  // Uncomment me when introducing the first migration for this slice
+  // version: Math.max(...Object.keys(clearOpportunitiesMigrations).map(Number)),
+  // migrate: createMigrate(clearOpportunitiesMigrations, { debug: false }),
 }
 
 const nftPersistConfig = {
   key: 'nft',
   storage: localforage,
-  version: Math.max(...Object.keys(clearNftsMigrations).map(Number)),
-  migrate: createMigrate(clearNftsMigrations, { debug: false }),
+  version: 0,
+  // Uncomment me when introducing the first migration for this slice
+  // version: Math.max(...Object.keys(clearNftsMigrations).map(Number)),
+  // migrate: createMigrate(clearNftsMigrations, { debug: false }),
 }
 
 const snapshotPersistConfig = {
   key: 'snapshot',
   storage: localforage,
-  version: Math.max(...Object.keys(clearSnapshotMigrations).map(Number)),
-  migrate: createMigrate(clearSnapshotMigrations, { debug: false }),
+  version: 0,
```

Comment:
feels like we will be using version 0 for an actual migration when we add the first one. what about just populating the initial migration across the board so we don't need all of the comments and call it a day?